### PR TITLE
CHANGE example urls to netlify urls

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: we switched to netlify for examples builds

## What I did
CHANGE links to ref netlify

### CODE REVIEW

**Comment by ndelangen** [comment]
This needs https://github.com/storybookjs/storybook/pull/9585 to be merged into master, and wait for the master deployment to be completed successfully

File: docs/src/pages/basics/live-examples/index.md [hunk 1]
```diff
@@ -5,55 +5,47 @@ title: 'Live Examples'
...
-- [Angular](https://storybooks-angular.netlify.com/)
-- [Polymer](https://storybooks-polymer.netlify.com/)
-- [Mithril](https://storybooks-mithril.netlify.com/)
-- [Marko](https://storybooks-marko.netlify.com/)
-- [HTML](https://storybooks-html.netlify.com/)
-- [Svelte](https://storybooks-svelte.netlify.com/)
-- [Riot](https://storybooks-riot.netlify.com/)
-- [Ember](https://storybooks-ember.netlify.com/)
-- [Preact](https://storybooks-preact.netlify.com/)
+- [React Official](https://storybookjs.netlify.com/official-storybook/)
```
Comment (shilman) [inline docs/src/pages/basics/live-examples/index.md]: @ndelangen these links don't work. is there another PR we need to patch back to `master`?

Comment (ndelangen) [reply docs/src/pages/basics/live-examples/index.md]: Going to investigate

Comment (ndelangen) [reply docs/src/pages/basics/live-examples/index.md]: Going to fix this asap:
https://app.netlify.com/sites/storybookjs/deploys/5e3a8b3054f52300084af24c

**Pull request merged**